vegasMode/main.c

```diff
@@ -202,13 +202,13 @@ static void TriggeredPeriodHandler
 )
 //--------------------------------------------------------------------------------------------------
 {
-    if ((value <= 0) || (value > 0xFFFFFFFF) || isnan(value))
+    if (isnan(value) || (value <= 0.0) || ((value * 1000.0) > UINT32_MAX))
```

Comment:
I just realized that there's another problem if the value is < 0.001, then `(uint32_t)(value * 1000.0)` will still be zero.  I think this would work, but it's kind of long.
```
if (isnan(value) || value < 0.0)
{
    LE_ERROR("Invalid period (%lf seconds)", value);
    return;
}

const double periodMs = value * 1000.0;
if (periodMs >UINT32_MAX)
{
    LE_ERROR("Period is too long (%lf seconds)", value);
    return;
}

const uint32_t periodMsInt = (uint32_t)periodMs;
if (periodMsInt == 0)
{
    LE_ERROR("Period can't be set to 0");
    return;
}

TriggeredModPeriod = perodMsInt;
```